### PR TITLE
Add hotel estimates

### PR DESCRIPTION
**Pull request by boconnell2**
### What

- Added hotel estimates to the SDK

### Why

- Added new estimates API endpoint

### SDK Release Checklist

- [x] Have you added an integration test for the changes?
- [ ] Have you built the package locally and made queries against it successfully?
- [x] Did you update the changelog?
- [x] Did you bump the package version?
- [ ] If endpoints were removed, did you manually remove the corresponding files? (this should be rare)
- [ ] For breaking changes, did you plan for the release of the new SDK versions and deploy the API to production?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,6 +21,8 @@
     "test": "mocha"
   },
   "dependencies": {
+    "@patch-technology/patch": "^1.18.0",
```
Comment (boconnell2) [inline package.json:24]: Should this say 1.19.0? I edited it directly but it seems like it auto corrected back to 1.18.0

File: README.md [hunk 2]
```diff
@@ -230,6 +246,7 @@ patch.projects.retrieveProjects().then((response) => console.log(response));
 ### Run the specs
 
 Before running the tests, make sure you set the test API key! Please use test API keys and not production ones, they usually start with `key_test_`.
+Additionally, make sure you navigate back to the `patch-node` file.
```
Comment (kleinjm) [inline README.md:249]:
```suggestion
Be sure you navigate back to the root `patch-node` directory to run the tests.
```

File: test/integration/estimates.test.js [hunk 3]
```diff
@@ -135,4 +135,20 @@ describe('Estimates Integration', function () {
...
+      city: 'New York',
+      region: 'New York',
+      star_rating: 5,
+      number_of_nights: 2,
+      number_of_rooms: 2
+    });
+    const estimate = createEstimateResponse.data;
+
+    expect(estimate.type).to.be.eq('hotel');
+    expect(estimate.mass_g).to.be.above(0);
```
Comment (kleinjm) [inline test/integration/estimates.test.js:151]: want to make this assertion greater than a certain value?

**Review by kleinjm — CHANGES_REQUESTED** [review]
nice work! Small blocker to remove the `yarn.lock` and just have the `package.lock` but otherwise LGTM

**Comment by boconnell2** [comment]
> nice work! Small blocker to remove the `yarn.lock` and just have the `package.lock` but otherwise LGTM

To confirm, it looks like the `package-lock.json` file is currently on the main branch. That one should stay right?

File: package.json (hunk 1 above)
Comment (rmody3) [reply package.json:24]: i actually dont think we need this dependency here at all? Seems weird that the package depending on an older version of itself

**Review by rmody3 — CHANGES_REQUESTED** [review]
Comparing to the [prior PR](https://github.com/patch-technology/patch-node/pull/59/files#) and others before:  it looks like you are missing the change to https://github.com/patch-technology/patch-node/pull/59/files#diff-e6d2e8d27919f843bb37cb10a5b6b5cc31e24a2f4bdcc9ae3626b23376cfb517L19

I think the reason you may not have had this is because you may have missed this [step](https://github.com/patch-technology/client-code-generation/blob/main/README.md#updating-existing-sdks) in the client-code generation: 

> Before running the generator, update the SDK version. Do so by simply changing the version number in the configs/<language>-config.json file. See [an example here](https://github.com/patch-technology/client-code-generation/pull/43/commits/e674f876ef3a978351d71e8110966af314c9af63).



like I did in this prior PR: https://github.com/patch-technology/client-code-generation/pull/57/files#diff-963d042286841a727b4a008883e075370ff4edb046b3405b35044d6d5ed02077


I think this is also the reason you have an internal dependency of the prior version in your `package.json` file`

Can chat IRL if you need additional help.

File: package.json (hunk 1 above)
Comment (boconnell2) [reply package.json]: I defer to the broader team's judgement here

**Review by rmody3 — APPROVED** [review]

**Pull request merged**